db/migrate/20240408102550_add_created_by.rb

```diff
@@ -0,0 +1,11 @@
+class AddCreatedBy < ActiveRecord::Migration[7.0]
+  def change
+    change_table :sales_logs do |t|
+      t.references :created_by, class_name: "User"
```

Comment:
this was previously created_by_id, are we calling it just created_by intentionally? Or does rails ignore this difference anyway

Reply:
I think cause we're using `.reference` to create this instead of `.column`, it actually ends up adding `created_by_id` to schema